sphinx/testing/fixtures.py

```diff
@@ -82,7 +82,7 @@ def app_params(request: Any, test_params: dict, shared_result: SharedResult,
     if test_params['shared_result']:
         if 'srcdir' in kwargs:
             msg = 'You can not specify shared_result and srcdir in same time.'
-            raise pytest.Exception(msg)
+            pytest.fail(msg)
```

Comment:
As you can see, using 'skip' was a bad idea since pytest.Exception never existed in the first place !